packages/yew-macro/src/props/prop.rs

```diff
@@ -82,6 +83,16 @@ impl Parse for Prop {
     }
 }
 
+fn validate_prop(expr: &Expr) -> syn::Result<()> {
+    match expr {
+        Expr::Block(_) | Expr::Lit(_) => Ok(()),
+        _ => Err(syn::Error::new_spanned(
+            expr,
+            format!("property value must be either a literal or enclosed in braces. Consider adding braces around your expression like this: {{ {} }}", expr.to_token_stream()),
+        )),
+    }
+}
+
```

Comment:
This is the actual code corresponding to the change (just so that it its easier to find in this PR)